ddp-workspace/projects/ddp-rgp/src/app/constants/study-messages.ts

```diff
@@ -152,7 +157,7 @@ export const studyMessagesConfiguration: StudyMessageConfiguration[] = [
       workflow.status === AcceptanceStatus.Accepted ||
       workflow.status === AcceptanceStatus.MoreInfoNeeded,
     additionalCondition: workflows =>
-      !!workflows.find(workflow => workflow.workflow === WorkflowKey.EnrollmentDate && workflow.status) &&
+      !workflows.find(workflow => workflow.workflow === WorkflowKey.EnrollmentDate && workflow.status) &&
```

Comment:
Wait. This is row #4b in the spreadsheet. This should be `AcceptanceStatusDate` instead of `EnrollmentDate` since it depends on that date field? Probably a copy-paste mistake on my part, sorry about that.
```suggestion
      !!workflows.find(workflow => workflow.workflow === WorkflowKey.AcceptanceStatusDate && workflow.status) &&
```